package.json

```diff
@@ -117,6 +117,7 @@
     "react-router": "^2.2.0",
     "react-router-redux": "^4.0.0",
     "redux": "^3.0.0",
+    "redux-form": "^5.3.1",
```

Comment:
It's stable enough to use and has improved features and better docs
http://redux-form.com/6.0.0-alpha.4/

For evaluation purposes it's also a good opportunity to try it out